### PR TITLE
[ACR-1579] Add machine-learning-training docs

### PR DESCRIPTION
**Pull request by zhoonit**
**Changes proposed in this PR:**
- Add machine-learning-training docs
- Update overview.md

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

File: docs/application/native/guides/machine-learning/machine-learning-training.md [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+2. include the following features in your `tizen-manifest.xml` file:
+
+   ```xml
+   <feature name="http://tizen.org/feature/machine_learning">true</feature>
+   <feature name="http://tizen.org/feature/machine_learning.training">true</feature>
+   ```
+
+3. In case of saving/loading model files from the outside of the application's own resources, the application has to request permission by adding following privileges to the `tizen-manifest.xml` file:
+
+  ```xml
```
Comment (jijoongmoon) [inline docs/application/native/guides/machine-learning/machine-learning-training.md]: need to align

File: docs/application/native/guides/machine-learning/machine-learning-training.md [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+&#xfeff;                    |                    | lecun_normal  | LeCun Normal Initialization
+&#xfeff;                    |                    | lecun_uniform | LeCun Uniform Initialization
+&#xfeff;                    |                    | xavier_normal | Xavier Normal Initialization
+&#xfeff;                    |                    | xavier_uniform| Xavier Uniform Initialization
+&#xfeff;                    |                    | he_normal     | He Normal Initialization
+&#xfeff;                    |                    | he_uniform    | He Uniform Initialization
+&#xfeff;                    | bias_initializer   | (categorical) | Bias initializer, same category as `weight_initializer`
+&#xfeff;                    | weight_decay       | (categorical) | Weight decay. Currently, only l2norm is supported
+&#xfeff;                    |                    | l2norm        | l2norm weight decay
+&#xfeff;                    | weight_decay_lambda| (float)       | weight decay rate
```
Comment (jijoongmoon) [inline docs/application/native/guides/machine-learning/machine-learning-training.md]: remove weight decay

File: docs/application/native/guides/machine-learning/machine-learning-training.md [hunk 3]
```diff
@@ -0,0 +1,460 @@
...
+Normalization = true
+
+[outputlayer]
+Type = fully_connected
+Unit = 2    # Output Layer Dimension ( = Weight Width )
+bias_initializer = zeros
+weight_initializer = xavier_uniform
+Activation = sigmoid  # activation : sigmoid, softmax
+Weight_Decay = l2norm
+weight_Decay_Lambda = 0.005
```
Comment (jijoongmoon) [inline docs/application/native/guides/machine-learning/machine-learning-training.md]: remove weight decay

File: docs/application/native/guides/machine-learning/machine-learning-training.md (hunk 1 above)
Comment (zhoonit) [reply docs/application/native/guides/machine-learning/machine-learning-training.md]: Done

File: docs/application/native/guides/machine-learning/machine-learning-training.md (hunk 3 above)
Comment (zhoonit) [reply docs/application/native/guides/machine-learning/machine-learning-training.md]: Done. updated to weight_regularizer

File: docs/application/native/guides/machine-learning/machine-learning-training.md (hunk 2 above)
Comment (zhoonit) [reply docs/application/native/guides/machine-learning/machine-learning-training.md]: Done. updated to weight_regularizer
